graphql_compiler/cost_estimation/statistics.py

```diff
@@ -82,6 +82,35 @@ def get_distinct_field_values_count(self, vertex_name, field_name):
         """
         return None
 
+    def get_field_quantile(self, vertex_name, field_name, fraction):
+        """Return the field value that's larger than a given fraction of all values.
```

Comment:
nit: Return the smallest field value that's larger than a given fraction of all values.